constitution.tex

```diff
@@ -755,25 +758,32 @@ \subsection{Modifying the Constitution}
 	\item Each candidate is given a minimum of twenty-four hour period to accept or decline the nomination.
 	\item A list of all nominees who have accepted is presented to E-Board for approval.
 	      This E-Board Meeting is closed to E-Board Members, RTPs, and members with explicit invitation from E-Board.
-	\item If an E-Board Member is a candidate for the office in discussion, the member is absent and their vote is abstained.
+	\item If an E-Board Member is a candidate for the office in discussion, the member is absent, and their vote is abstained.
 	      An E-Board Vote, as described in \ref{Executive Board Vote}, is taken to determine whether the nominations of the RTP(s) are accepted.
 \end{enumerate}
+Giving an extension to a previous Active Member is the same as starting a selection process for a new RTP and should also follow this process.
 
 \asubsubsection{Qualifications of a Root Type Person}
-Candidates must be Active Members.
+Candidates must be Active Members or previous Active Members who are given an extension by the RTPs.
 
 \asubsubsection{Prior Root Type Persons}
-Prior RTPs are those members who are no longer current Active Members and have not been granted an extension by the current RTPs.
-Prior RTPs are not guaranteed access to the current root passwords and other authentication tokens.
+Prior RTPs are any members who were previously an RTP and are not currently an Active Member or on an extension given by the RTPs.
+Prior RTPs are not guaranteed access to any elevated permissions to any systems or networks, including but not limited to the RTP LDAP role, Vault, or the root password.
```

Comment:
why specify Vault? Authentication tokens and root passwords already covered this beyond any specific configuration or technology

(to quibble, we don't use hashicorp's vault, we have vaultwarden, and 5 years ago we had passbolt)